services/date.d.ts

```diff
@@ -0,0 +1,17 @@
+import Service from '@ember/service';
```

Comment:
Why are you manually building a d.ts file? We should be having the ts compiler produce the declaration file using `"declaration": true`.

Reply:
If it's not part of the package itself, how do consuming apps take advantage?

Reply:
It's created when we transpile the ts, so long as we specify `"declaration": true` in the tsconfir.json and we use a `prepublishOnly` in package.json so that the build occurs before we publish.